compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIEPackAndTranspose.cpp

```diff
@@ -61,10 +64,11 @@ static FailureOr<PackConfig> getPackConfig(
     // 0 is used when unpack is empty
     config.unpackEmpty = {0};
     config.innerPerm = {{1, 0}};
-    config.outerPerm = {{0, 1}};
+    config.outerPerm = {{0, 1, 2}};
```

Comment:
I believe this path is run for both matmul and batch matmul? If yes, I am wondering why it is okay add an extra permutation for the matmul. Perhaps the reason is that {0,1,2} is identity or no permutation at all? in which case we can drop this line all together?

Reply:
Oh no no - I am not sure if that claim would be right. So, I'll update this logic based on the `numLoops`/`numDims`/etc.

Currently just added the infra that just targets the batch_matmul - working on cleaning it up - should be complete by tomorrow.

We can currently review of the tile sizes and the packing config I'm using particularly for batch_matmul case.